games-emulation/gw_libretro/gw_libretro-1.6.2_20180125.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="A libretro core for Game & Watch simulators"
+DESCRIPTION="gw-libretro is a libretro core that runs Game & Watch simulators. \
+It runs simulators converted from source code. The converted games are \
+available at libretro's buildbot site. Unpack the zip and read the README.txt \
+for instructions."
+HOMEPAGE="http://bot.libretro.com/assets/cores/Handheld%20Electronic%20Game/"
+COPYRIGHT="2013-2018 the libretro team"
+LICENSE="Zlib
+	CC0-1.0"
```

Comment:
"CC0" is shipped with Haiku under the name ["Public Domain"](https://git.haiku-os.org/haiku/plain/data/system/data/licenses/Public%20Domain). Also, as stated on Github:

> The SNES controller, the hand cursor, and the Boxy Bold font used in gw-libretro are released under the CC0.

But I don't see any resources being packaged with the port (at least in `INSTALL()`). Can you please confirm if any resources was embedded within the port?

```diff
-	CC0-1.0"
+	Public Domain"
```

Reply:
They're embedded withing the core yes. I will replace the license by PD. Thanks!